tests/test_client_functional.py

```diff
@@ -3746,6 +3746,72 @@ async def handler(request: web.Request) -> NoReturn:
         await client.get("/", headers=headers)
 
 
+async def test_netrc_auth_from_env(
+    aiohttp_client: AiohttpClient, netrc_default_contents: pathlib.Path
+) -> None:
```

Comment:
Here's how to inject fixtures when their return values aren't actually in use:
```suggestion
@pytest.mark.usefixtures('netrc_default_contents')
async def test_netrc_auth_from_env(aiohttp_client: AiohttpClient) -> None:
```

Reply:
Yeah I always forget about that one. I should know better. That's what happens when you are tired though